nodes/common/pulp_node/reports.py

```diff
@@ -27,7 +13,7 @@ class RepositoryReport(object):
 
     # actions
     PENDING = 'pending'
-    CANCELLED = 'cancelled'
+    CANCELLED = 'canceled'
```

Comment:
This change may also be unsafe, as this sounds like it is used in the Node report API.